vite.config.ts

```diff
@@ -22,15 +27,35 @@ export default defineConfig({
 						'**/*.css',
 						'**/*.test.{js,jsx,ts,tsx}',
 						'**/__*.*',
-						// This is for server-side utilities you want to colocate next to
-						// your routes without making an additional directory.
-						// If you need a route that includes "server" or "client" in the
-						// filename, use the escape brackets like: my-route.[server].tsx
+						// This is for server-side utilities you want to colocate
+						// next to your routes without making an additional
+						// directory. If you need a route that includes "server" or
+						// "client" in the filename, use the escape brackets like:
+						// my-route.[server].tsx
 						'**/*.server.*',
 						'**/*.client.*',
 					],
 				})
 			},
 		}),
+		process.env.SENTRY_AUTH_TOKEN ??
+			sentryVitePlugin({
+				disable: MODE !== 'production',
+				authToken: process.env.SENTRY_AUTH_TOKEN,
+				org: process.env.SENTRY_ORG,
+				project: process.env.SENTRY_PROJECT,
+				release: {
+					name: process.env.COMMIT_SHA,
```

Comment:
good job on setting the release name based on the commit sha. It will help people that are on a monorepo with pnpm. indeed when using a custom dockerfile with pnpm deploy, there is no ./git folder and sentry fails to automatically generate a release name